cdap-ui/app/cdap/components/WizardModal/WizardModal.scss

```diff
@@ -16,15 +16,19 @@
 
 @import '../../styles/variables.scss';
 $wizard-header-color: #464a57;
-$wizard-modal-margin-top: 180px;
+$wizard-modal-height: 85vh;
+$wizard-modal-width: 60vw;
+$wizard-modal-margin-top: 12.5vh;
+$wizard-modal-body-max-height: 360px;
+$wizard-modal-max-width: 1050px;
 
 .wizard-modal {
   overflow-x: hidden;
   &.modal-dialog {
-    height: 65vh;
-    width: 60vw;
+    height: $wizard-modal-height;
```

Comment:
I think this is actually got reversed... on line 50, the `.modal-body` have max-height while the `.modal-dialog` is specific height.. That seems a little weird